### ISSUE TITLE
[wasm] Wasm.Build.Tests failing

### ISSUE THREAD

**Issue by radical** [dotnet/runtime#53405]
- Wasm.Build.Tests are failing with 

```
    Wasm.Build.Tests.WasmBuildAppTest.AsyncMain [FAIL]
      System.TypeInitializationException : The type initializer for 'Wasm.Build.Tests.BuildTestBase' threw an exception.
      ---- System.TypeLoadException : Could not load type 'System.IO.DirectoryInfo' from assembly 'System.Runtime, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
      Stack Trace:
        /_/src/tests/BuildWasmApps/Wasm.Build.Tests/BuildTestBase.cs(123,0): at Wasm.Build.Tests.BuildTestBase.ConfigWithAOTData(Boolean aot)
        /_/src/tests/BuildWasmApps/Wasm.Build.Tests/WasmBuildAppTest.cs(19,0): at Wasm.Build.Tests.WasmBuildAppTest.MainMethodTestData(Boolean aot, RunHost host)
        ----- Inner Stack Trace -----
```

[This](https://github.com/dotnet/runtime/commit/44f050acc0f7791d6cd7ac772945444912bcf299) commit might be the one responsible:

```
commit 44f050acc0f7791d6cd7ac772945444912bcf299
Author: Adam Sitnik <adam.sitnik@gmail.com>
Date:   Thu May 27 12:56:30 2021 +0200

    Fold System.IO.FileSystem into CoreLib (#53231)

    * move Queue<T> to System.Private.CoreLib

    * fold System.IO.FileSystem into System.Private.CoreLib
```

Looking at the checks on the PR #53231, `Wasm.Build.Tests` were failing.

This is a build before the PR was merged - https://dev.azure.com/dnceng/public/_build/results?buildId=1159218&view=logs&s=96ac2280-8cb4-5df5-99de-dd2da759617d

And this is after: https://dev.azure.com/dnceng/public/_build/results?buildId=1159354&view=logs&j=108d2c4a-8a62-5a58-8dad-8e1042acc93c&t=568f884b-cc12-5fd3-e7fe-790b5ac403f4

@adamsitnik please correct me if I got this wrong

**Comment by adamsitnik** [comment]
@radical please excuse me for that. I've merged a fix in #53410

**Comment by radical** [comment]
@adamsitnik no worries. Thanks for the quick fix. I have opened https://github.com/dotnet/runtime/pull/53433 to re-enable the tests.
